### PR TITLE
Auto fix lint warnings

### PR DESCRIPTION
**Pull request by DimalChandrasiri**
### Purpose
This PR will add lint fixes which were done using the `--fix` flag where eslint autofixes simple lint errors.

### CODE REVIEW

**Review by yasinmiran — APPROVED** [review]

File: apps/console/src/features/applications/components/help-panel/overview.tsx [hunk 1]
```diff
@@ -103,7 +103,7 @@ export const HelpPanelOverview: FunctionComponent<HelpPanelOverviewPropsInterfac
             {
                 !isOIDCConfigsLoading ? (
                     <Grid>
-                        {/*  {
+                        { /*  {
```
Comment (jeradrutnam) [inline apps/console/src/features/applications/components/help-panel/overview.tsx:106]: Why do we need this line. Shall we check and add a TODO if we are going to re- enable it near future

File: apps/console/src/features/applications/components/modals/certificate-form-field-modal.tsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ import { DisplayCertificate, TestableComponentInterface } from "@wso2is/core/mod
 import { CertificateManagementUtils } from "@wso2is/core/utils";
 import {
     Certificate as CertificateDisplay,
-    GenericIcon} from "@wso2is/react-components";
+    GenericIcon } from "@wso2is/react-components";
```
Comment (jeradrutnam) [inline apps/console/src/features/applications/components/modals/certificate-form-field-modal.tsx:23]: Can go in to single line

File: apps/console/src/features/applications/components/help-panel/overview.tsx (hunk 1 above)
Comment (DimalChandrasiri) [reply apps/console/src/features/applications/components/help-panel/overview.tsx]: Fixed with commit 2db59e4

File: apps/console/src/features/applications/components/modals/certificate-form-field-modal.tsx (hunk 2 above)
Comment (DimalChandrasiri) [reply apps/console/src/features/applications/components/modals/certificate-form-field-modal.tsx]: Resolved with 2db59e4

**Review by jeradrutnam — APPROVED** [review]

**Pull request merged**